src/main/kotlin/tech/thatgravyboat/craftify/ui/PositionEditorScreen.kt

```diff
@@ -37,8 +36,8 @@ class PositionEditorScreen : WindowScreen(ElementaVersion.V1) {
 
         if (button == 0) {
             this@onMouseDrag.constrain {
-                x = (MathHelper.clamp_float(this@onMouseDrag.getLeft() + mouseX - clickPos!!.first, 0f, UResolution.scaledWidth - 150f)).pixels()
-                y = (MathHelper.clamp_float(this@onMouseDrag.getTop() + mouseY - clickPos!!.second, 0f, UResolution.scaledHeight - 50f)).pixels()
+                x = (clamp(this@onMouseDrag.getLeft() + mouseX - clickPos!!.first, 0f, UResolution.scaledWidth - 150f)).pixels()
```

Comment:
I didnt realize when I wrote this that I still was using math helper but this should use kotlins coerceIn